lumicks/pylake/kymotracker/tests/test_refinement.py

```diff
@@ -82,6 +81,22 @@ def test_refinement_track(loc, inv_sigma=0.3):
     np.testing.assert_allclose(track.coordinate_idx, loc, rtol=1e-2)
 
 
+@pytest.mark.parametrize("loc", [25.3, 25.5, 26.25, 23.6])
+def test_refinement_with_background(loc, inv_sigma=0.3):
```

Comment:
why is `inv_sigma` a parameter with a default value? "who" can actually provide the value here?

Reply:
It shouldn't be. Removed now.